calico/felix/actor.py

```diff
@@ -218,6 +227,10 @@ def _step(self):
             batch = self._start_msg_batch(batch)
             assert batch is not None, "_start_msg_batch() should return batch."
             results = []  # Will end up same length as batch.
+
+            # Will store the logs for the messages from this batch.
```

Comment:
This variable doesn't store logs, it stores the UUIDs from the messages.  I'm not sure why it's needed, don't we already have a list of the messages in the batch?